config/src/main/java/org/springframework/security/config/annotation/web/configurers/CsrfCustomizer.java

```diff
@@ -0,0 +1,57 @@
+package org.springframework.security.config.annotation.web.configurers;
```

Comment:
Please add a copyright header.

If you run `./gradlew format && ./gradlew check` it will catch things like this.